resources/report-templates/standard-template/jinja2/minimum-needs.html

```diff
@@ -1,3 +1,4 @@
+{% if needs %}
```

Comment:
Safely create empty string if the layer doesn't support minimum needs breakdown (applied for population)